post_processing_scripts/VaryTempWithHeight.py

```diff
@@ -89,23 +98,24 @@ def execute(self, data):
 
                 # If we've found our line
                 if match is not None:
+                    output_line += ';TYPE:CUSTOM\n'
                     # Grab the z value
                     new_z = float(match.groups()[0])
 
                     # If our z value has changed
                     if new_z != z:
-                        z = new_z
-
-                        # Determine new temperature
-                        new_temp = int(z / height_inc) * temp_inc
-                        new_temp = start_temp - new_temp
-
-                        # If we hit a spot where we need to change the
-                        # temperature, then write the gcode command
-                        if new_temp < current_temp:
-                            current_temp = new_temp
-                            output_line += ';TYPE:CUSTOM\n'
-                            output_line += 'M104 S%d\n' % new_temp
+                        z = new_z - feet_height
+                        if z == 0:
```

Comment:
Will z ever actually be exactly 0? Since it is a float, and people use varying z increment values, I don't think this will work.